1-Draft/RFC0002-Generalized-Splatting.md

```diff
@@ -130,6 +131,15 @@ If '@' is the splatting operator,
 adding the '?' is suggestive of being more permissive,
 much like the C# '?.' member access operator.
 
+If parameter values are passed explicitly in addition to the relaxed splatting operator,
+those values would take precedence over anything in the splatted hashtable:
```

Comment:
Shall we support this behavior only for `@?` syntax, or support it for the existing `@` splatting as well?
Note that, today this will fail with the existing splatting.
```
PS:173> $p = @{ Name = "Name"; Path = "Path" }
PS:175> bar @p -Name fff
bar : Cannot bind parameter because parameter 'Name' is specified more than once. To provide multiple values to parameters that can accept multiple values, use the array syntax. For example, "-parameter value1,value2,value3".
At line:1 char:8
+ bar @p -Name fff
+        ~~~~~
+ CategoryInfo          : InvalidArgument: (:) [bar], ParameterBindingException
+ FullyQualifiedErrorId : ParameterAlreadyBound,bar
```

Reply:
@PowerShell/powershell-committee as discussed, this is only for relaxed splatting